tools/task-runner/runner

```diff
@@ -78,18 +13,20 @@ const chalk = require('chalk');
 const figures = require('figures');
 const uniq = require('lodash.uniq');
 
+// name of the tasks directory
 const tasksIdentifier = '__tasks__';
```

Comment:
Why not name this `tasksDirectory` , then you don't need the comment as it is self explanatory ?